core/src/main/scala/com/databricks/labs/remorph/parsers/FunctionBuilder.scala

```diff
@@ -142,7 +148,9 @@ object FunctionBuilder {
     case "ISDATE" => Some(FixedArity(1))
     case "ISDESCENDANTOF" => Some(FixedArity(1))
     case "ISJSON" => Some(VariableArity(1, 2))
-    case "ISNULL" => Some(FixedArity(2))
+    // Though ISNULL is FixedArity, it is one argument in Snowflake (and others) but two arguments in TSql
```

Comment:
This difference isn't addressed by the code bellow (we get a `VariableArity` even in the case of Snowflake).